docs/en/sql-reference/table-functions/remote.md

```diff
@@ -68,27 +68,20 @@ Multiple addresses can be comma-separated. In this case, ClickHouse will use dis
 example01-01-1,example01-02-1
 ```
 
-Part of the expression can be specified in curly brackets. The previous example can be written as follows:
+<a name="globs-in-addresses"></a>
 
-``` text
-example01-0{1,2}-1
-```
-
-Curly brackets can contain a range of numbers separated by two dots (non-negative integers). In this case, the range is expanded to a set of values that generate shard addresses. If the first number starts with zero, the values are formed with the same zero alignment. The previous example can be written as follows:
+**Globs in Addresses**
 
-``` text
-example01-{01..02}-1
-```
+Patterns in curly brackets `{ }` are used to generate a set of shards and to specify replicas. If there are multiple pairs of curly brackets, then the direct product of the corresponding sets is generated. 
+The following pattern types are supported.
 
-If you have multiple pairs of curly brackets, it generates the direct product of the corresponding sets.
-
-Addresses and parts of addresses in curly brackets can be separated by the pipe symbol (\|). In this case, the corresponding sets of addresses are interpreted as replicas, and the query will be sent to the first healthy replica. However, the replicas are iterated in the order currently set in the [load_balancing](../../operations/settings/settings.md#settings-load_balancing) setting. This example specifies two shards that each have two replicas:
-
-``` text
-example01-{01..02}-{1|2}
-```
+- {*a*,*b*} - Any number of variants separated by a comma. The pattern is replaced with *a* in the first shard address and it is replaced with *b* in the second shard address and so on. For instance, `example0{1,2}-1` generates addresses `example01-1` and `example02-1`.
+- {*n*..*m*} - A range of numbers. This pattern generates shard addresses with incrementing indices from *n* to *m*. `example0{1..2}-1` generates `example01-1` and `example02-1`.
+- {*0n*..*0m*} - A range of numbers with leading zeroes. This modification preserves leading zeroes in indices. The pattern `example{01..03}-1` generates `example01-1`, `example02-1` and `example03-1`.
+- {*a*|*b*} - Any number of variants separated by a `|`. The pattern specifies replicas. For instance, `example01-{1|2}` generates replicas `example01-1` and `example01-2`.
 
-The number of addresses generated is limited by a constant. Right now this is 1000 addresses.
+The query will be sent to the first healthy replica. However, the replicas are iterated in the order currently set in the [load_balancing](../../operations/settings/settings.md#settings-load_balancing) setting. 
```

Comment:
Not correct, this settings is not used. Set of replicas is iterated in random order, which is different each time the query is made.

Reply:
Not sure, what you mean.
Tried two settings and saw the difference.

`SELECT * FROM remote('127.0.0.{2|3|4}', db.remote_engine_table) SETTINGS load_balancing = 'in_order';`
Every time returns addresses in the same order.
```
Code: 210, e.displayText() = DB::NetException: Connection refused (127.0.0.2:9000) (version 21.8.1.7290 (official build))
Code: 210, e.displayText() = DB::NetException: Connection refused (127.0.0.3:9000) (version 21.8.1.7290 (official build))
Code: 210, e.displayText() = DB::NetException: Connection refused (127.0.0.4:9000) (version 21.8.1.7290 (official build))
```

With `random` setting the order in the output differs every time.
SELECT * FROM remote('127.0.0.{2|3|4}', db.remote_engine_table) SETTINGS load_balancing = 'random';

Reply:
Thought it was about url table function since this pr is named as 'description of patterns for url table function'. For remote it is correct.

Reply:
```suggestion
The query will be sent to the first healthy replica. However, for `remote` the replicas are iterated in the order currently set in the [load_balancing](../../operations/settings/settings.md#settings-load_balancing) setting. 
```